<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PoliciesEdit</class>
 <widget class="QFrame" name="PoliciesEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>635</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="0" colspan="8">
    <widget class="QPlainTextEdit" name="errors"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Policy name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="7">
    <widget class="QLineEdit" name="name"/>
   </item>
   <item row="2" column="0" colspan="8">
    <widget class="QGroupBox" name="groupBox">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="6" column="0" colspan="2">
       <widget class="QTextEdit" name="freeText"/>
      </item>
      <item row="8" column="1">
       <widget class="QPushButton" name="newRule">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 170, 0);
selection-background-color: rgb(255, 170, 0);</string>
        </property>
        <property name="text">
         <string>Add new policy rule</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5" rowspan="7" colspan="3">
       <widget class="QTableWidget" name="rules">
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <property name="gridStyle">
         <enum>Qt::NoPen</enum>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Policy rules</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="8" column="5">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>150</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="editorSelector">
        <property name="text">
         <string>Edi&amp;tor</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Rule name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="ruleName"/>
      </item>
      <item row="2" column="0" rowspan="4" colspan="3">
       <widget class="QFrame" name="editorFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="5" column="2">
          <widget class="QComboBox" name="validator"/>
         </item>
         <item row="2" column="2">
          <widget class="QComboBox" name="type"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Value:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Type:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Validator:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Field:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QLineEdit" name="value"/>
         </item>
         <item row="3" column="2">
          <widget class="QComboBox" name="field"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Rule policy:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="8" column="6">
       <widget class="QPushButton" name="deleteRule">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 0, 0);
selection-background-color: rgb(255, 0, 0);
</string>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="freeTextSelector">
        <property name="text">
         <string>Free &amp;text</string>
        </property>
       </widget>
      </item>
      <item row="8" column="7">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>150</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
     <zorder>rules</zorder>
     <zorder>editorFrame</zorder>
     <zorder>editorSelector</zorder>
     <zorder>freeTextSelector</zorder>
     <zorder>newRule</zorder>
     <zorder>deleteRule</zorder>
     <zorder>ruleName</zorder>
     <zorder>label</zorder>
     <zorder>freeText</zorder>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QDialogButtonBox" name="validation">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
